src/routes.tsx

```diff
@@ -79,6 +81,15 @@ export const routes: RouteObject[] = [
             element: <ProposalDetailsPage />,
             loader: proposalIdParamLoader,
           },
+          {
+            path: `validators`,
+            element: <ValidatorsPage />,
+          },
+          {
+            path: `validators/:address`,
+            element: <ValidatorDetailsPage />,
+            loader: addressParamLoader(),
```

Comment:
this allows hex addresses

Reply:
will remove this line and add TODO to add it later when API will use address as param. Turns out it is using entity_id which is not what we want in URL.